### ISSUE TITLE
Constant probability for object detection

### ISSUE THREAD

**Issue by xternalz** [akuxcw/GridMask#7]
Hi,

Thanks for this nice work. It seems that you are using a constant GridMask probability for object detection. The `set_prob` function only gets called when `cfg.GRID.PROB == 0`. Does that work better than linearly increasing the probability?

Thanks.

**Comment by akuxcw** [comment]
Hi, in our experiments, we didn't observe obvious promotion when using linearly schedule of the probability in detection. We think it may because the backbone already uses pre-trained weights in detection.

**Comment by xternalz** [comment]
Thanks.

**Comment by hedes1992** [comment]
> Hi, in our experiments, we didn't observe obvious promotion when using linearly schedule of the probability in detection. We think it may because the backbone already uses pre-trained weights in detection.

Hello, have you do experiments using official pretrained model from pytorch, but not the pretrained model with GridMask from imagenet? If only use official pretrained model, I guess the linearly schedule is more better? Thank you

**Comment by akuxcw** [comment]
Hello,
We always use the official pretrained model from pytorch, which means we use the same pretrained model for baseline and our results. We didn't use pretrained model with GridMask from imagenet. Because we want to show it is gridmask that can improve the result of detection instead of the pretrained model.

**Comment by hedes1992** [comment]
> Hello,
> We always use the official pretrained model from pytorch, which means we use the same pretrained model for baseline and our results. We didn't use pretrained model with GridMask from imagenet. Because we want to show it is gridmask that can improve the result of detection instead of the pretrained model.

Thanks for your reply.
Ok, so you mean we can use prob=0.7 directly in detectino training because the official model is good enough?(In table7 of the grid mask paper).

**Comment by akuxcw** [comment]
Yes.
